src/components/GenericInput.vue

```diff
@@ -15,6 +15,7 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
     @keydown="blurOnEnter"
     @blur="onChange($event.target.value)"
     :value="row[col.name]"
+    :disabled="col.enabled ? !col.enabled(row) : false"
```

Comment:
Could you explain what this is doing? Is it something that might be better suited to a computed property?

Reply:
This says "The `<input>` is disabled if there is a `enabled` function defined on this column in libconfig, and the `col.enabled` function says to disable.

This is a new key I added to libconfig to support the new window types. We need to be able to disable editing of some fields based on the choice of `window_definition_type`. For example, `spacing` only makes sense for Repeated Windows.

I'm happy to move that to a computed column. That would cut down on some repetition. I'll do that now.